webapp/src/main/webapp/src/app/pages/services/services.page.spec.ts

```diff
@@ -17,15 +17,14 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { ServicesPageComponent } from './services.page';
 
-describe('ServicesPageComponent', () => {
+xdescribe('ServicesPageComponent', () => {
```

Comment:
Could you please remove this from the commit as it is not related to the purpose of the change?